demo-server/templates/index-template.handlebars

```diff
@@ -76,6 +76,7 @@
         function initWidget(widget) {
           {{#if redirectless}}
           var config = {
+            // interactWith: PfAuthnWidget.INTERACT_WITH_USER_AUTHZ,
             // client_id: '', // redirectless requires this configuration
             // response_type: 'token', // redirectless requires this configuration
```

Comment:
We should let the developer know how to provide the user_code parameter to the endpoint (for the case where user just needs to confirm it). Maybe there should be a separate field in the `config` object for the user code.

Reply:
Sure, I also need to consume the provided property to create the proper user_authz url.